web-app/src/app/screens/FAQ.tsx

```diff
@@ -17,166 +15,190 @@ export default function FAQ(): React.ReactElement {
+          The main benefit is that having your feed in the database ensures more
+          rider-facing apps discover and use your data. Plus, the integration
+          with the Canonical GTFS Schedule Validator and GBFS validator means
+          your GTFS or GBFS feed will be checked before you submit them to a
+          trip-planner or navigation application. You’ll receive a detailed
+          error quality report, which reduces the amount of back and forth in
+          the process and allows you to submit high-quality data to give your
+          riders reliable, detailed information. Another benefit of our Database
+          is simply the access you have to search, download, and look at other
+          feeds to get examples on how to improve your own data, whether it is
+          GTFS Schedule, GTFS Realtime, or GBFS.
         </Typography>
         <Typography
           variant='h5'
           color='primary'
           sx={{ fontWeight: 700, mt: 5, mb: 1 }}
         >
-          What’s coming next?
+          I want to consume/analyze/display transport data, how can I use the
+          Mobility Database?
         </Typography>
         <Typography className='answer'>
-          The MobilityData team is working to add historical data and route and
-          stop visualizations to the Mobility Database.
+          Our API allows you to pull data from our database seamlessly. Since
+          our URLs are stable and checked for updates on a daily basis, data
+          doesn’t get dropped if an agency’s website is down, or if the link
+          expires.
           <br /> <br />
+          Detailed instructions for installing and implementing our API can be
+          found on&#20;
           <Button
             variant='text'
             className='inline line-start'
-            href={'https://mobilitydata.org/roadmaps/'}
+            href={
+              'https://mobilitydata.github.io/mobility-feed-api/SwaggerUI/index.html'
+            }
+            rel='noreferrer'
+            target='_blank'
+          >
+            Swagger.
+          </Button>
+          In addition to searching the database or using our API to pull data,
+          you’re able to&#20;
+          <Button
+            variant='text'
+            className='inline line-start'
+            href={'https://files.mobilitydatabase.org/feeds_v2.csv'}
             rel='noreferrer'
             target='_blank'
             endIcon={<OpenInNewIcon />}
           >
-            You can add ideas and vote on our current roadmap
+            download GTFS and GTFS Realtime feeds via the spreadsheet here
+          </Button>
+          <br /> <br />
+          GBFS feeds can be downloaded via a spreadsheet with the&#20;
+          <Button
+            variant='text'
+            className='line-start inline'
+            href={
+              'https://github.com/MobilityData/gbfs?tab=readme-ov-file#systems-catalog---systems-implementing-gbfs'
```

Comment:
@emmambd Suggestion to link to the catalog itself.

```suggestion
              'https://github.com/MobilityData/gbfs/blob/master/systems.csv'
```